WalletWasabi.Fluent/Views/AddWallet/Create/ConfirmRecoveryWordsView.axaml

```diff
@@ -12,29 +12,81 @@
              x:CompileBindings="True"
              x:Class="WalletWasabi.Fluent.Views.AddWallet.Create.ConfirmRecoveryWordsView">
   <c:ContentArea Title="{Binding Title}"
+                 Caption="{Binding CurrentWord.Index, StringFormat='Click the recovery word #{0}', FallbackValue=' '}"
                  EnableCancel="{Binding EnableCancel}"
                  EnableBack="{Binding EnableBack}"
-                 EnableNext="True" NextContent="Continue"
-                 EnableSkip="{Binding IsSkipEnable}"
-                 IsBusy="{Binding IsBusy}">
-    <ScrollViewer HorizontalAlignment="Stretch" VerticalAlignment="Center" MaxWidth="350">
-      <ItemsControl Items="{Binding ConfirmationWords}">
-        <i:Interaction.Behaviors>
-          <behaviors:FocusFirstTextBoxInItemsControlBehavior />
-        </i:Interaction.Behaviors>
+                 EnableNext="True"
+                 NextContent="Continue"
+                 EnableSkip="{Binding IsSkipEnabled}"
+                 IsBusy="{Binding IsBusy}"
+                 ScrollViewer.VerticalScrollBarVisibility="Disabled">
+    <Grid x:Name="Layout">
+      <Grid.RowDefinitions>
+        <RowDefinition MinHeight="70" Height="*" />
+        <RowDefinition Height="16" />
+        <RowDefinition Height="Auto" />
+        <RowDefinition Height="*" />
+        <RowDefinition Height="20" />
+      </Grid.RowDefinitions>
+
+      <ItemsControl Items="{Binding AvailableWords}">
+        <ItemsControl.ItemsPanel>
+          <ItemsPanelTemplate>
+            <WrapPanel />
+          </ItemsPanelTemplate>
+        </ItemsControl.ItemsPanel>
         <ItemsControl.ItemTemplate>
-          <DataTemplate>
-            <DockPanel>
-              <Label Content="{Binding Index, StringFormat=Recovery word {0}, Mode=OneWay}" DockPanel.Dock="Top" />
-              <TextBox Watermark="Type in the word" Classes="hasCheckMarkBoolean" validation:CheckMarkStatus.IsEnabled="{Binding IsConfirmed}" Text="{Binding Input}" IsEnabled="{Binding !IsConfirmed}" DockPanel.Dock="Top">
-                <i:Interaction.Behaviors>
-                  <behaviors:FocusNextItemBehavior IsFocused="{Binding !IsConfirmed}" />
-                </i:Interaction.Behaviors>
-              </TextBox>
-            </DockPanel>
+          <DataTemplate x:DataType="create:RecoveryWordViewModel">
+            <ToggleButton IsChecked="{Binding IsSelected}"
+                          IsEnabled="{Binding IsEnabled}">
+              <ToggleButton.Styles>
+                <Style Selector="ToggleButton[IsChecked=True] /template/ c|TagControl">
+                  <Setter Property="Foreground" Value="{DynamicResource SystemAccentColor}" />
+                  <Setter Property="BorderBrush" Value="{DynamicResource SystemAccentColor}" />
+                </Style>
```

Comment:
This is confusing. The blue color really gets the attention although exactly the opposite would be the goal.

I would just remove it, and have the original colors. And meanwhile, I would make the disabled state more faded out with a 0.6 opacity. And also I would rather play with the `IsEnabled` property. Like whenever the word is confirmed I would just disable the button.

```xml
<ToggleButton.IsEnabled>
  <MultiBinding Converter="{x:Static BoolConverters.And}">
    <Binding Path="IsEnabled"/>
    <Binding Path="IsConfirmed" Converter="{x:Static BoolConverters.Not}"/>
  </MultiBinding>
</ToggleButton.IsEnabled>
```

The result would be a much cleaner view. And also fixed the weird behavior when the blue items are clickable but nothing happens.

So instead of this:
![image](https://user-images.githubusercontent.com/16364053/219024688-a32b79ab-1bda-48f3-a676-28a01289f828.png)
![image](https://user-images.githubusercontent.com/16364053/219024634-cca7b45f-4e2f-477b-a6c3-61544c958712.png)

It would look like this:
![image](https://user-images.githubusercontent.com/16364053/219024960-8cd2713f-b9f3-4147-8195-73a68feebd1f.png)
![image](https://user-images.githubusercontent.com/16364053/219024994-5d006892-3dda-4dab-9831-94cbd28b32fe.png)